napari/utils/_tests/test_conda.py

```diff
@@ -0,0 +1,23 @@
+import sys
+from unittest.mock import patch
+
+import pytest
+
+from napari.utils.conda import is_conda_package
+
+
+@pytest.mark.parametrize(
+    "pkg_name,expected",
+    [
+        ("some-package", True),
+        ("some-other-package", False),
+    ],
```

Comment:
```suggestion
    [
        ("some-package", True),
        ("some-other-package", False),
        ("some-package-other", False),
        ("other-some-package", False),
        ("package", False),
        ("some", False),
    ],
```

Use prefixed/suffixed names of package